registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java

```diff
@@ -1,697 +1,713 @@
+		abisReqEntity.setBioRefId(abisRequestDto.getBioRefId());
+		abisReqEntity.setCrBy(abisRequestDto.getCrBy());
+		if (abisRequestDto.getCrDtimes() == null) {
+			abisReqEntity.setCrDtimes(LocalDateTime.now(ZoneId.of("UTC")));
+		} else {
+			abisReqEntity.setCrDtimes(abisRequestDto.getCrDtimes());
+		}
+		abisReqEntity.setIsDeleted(false);
+		abisReqEntity.setLangCode(abisRequestDto.getLangCode());
+		abisReqEntity.setRefRegtrnId(abisRequestDto.getRefRegtrnId());
+		abisReqEntity.setReqBatchId(abisRequestDto.getReqBatchId());
+		abisReqEntity.setReqText(abisRequestDto.getReqText());
+		abisReqEntity.setRequestDtimes(abisRequestDto.getRequestDtimes());
+		abisReqEntity.setRequestType(abisRequestDto.getRequestType());
+		abisReqEntity.setStatusCode(abisRequestDto.getStatusCode());
+		abisReqEntity.setStatusComment(abisRequestDto.getStatusComment());
+		abisReqEntity.setUpdBy(abisRequestDto.getUpdBy());
+		abisReqEntity.setUpdDtimes(LocalDateTime.now(ZoneId.of("UTC")));
+		return abisReqEntity;
+
+	}
+
+	private AbisResponseEntity convertAbisResponseDtoToAbisResponseEntity(AbisResponseDto abisResponseDto) {
+		AbisResponseEntity abisResponseEntity = new AbisResponseEntity();
+		AbisResponsePKEntity abisResponsePKEntity = new AbisResponsePKEntity();
+		abisResponsePKEntity.setId(abisResponseDto.getId());
+		abisResponseEntity.setId(abisResponsePKEntity);
+		abisResponseEntity.setAbisRequest(abisResponseDto.getAbisRequest());
+		abisResponseEntity.setRespText(abisResponseDto.getRespText());
+		abisResponseEntity.setStatusCode(abisResponseDto.getStatusCode());
+		abisResponseEntity.setStatusComment(abisResponseDto.getStatusComment());
+		abisResponseEntity.setLangCode(abisResponseDto.getLangCode());
+		abisResponseEntity.setCrBy(abisResponseDto.getCrBy());
+		if (abisResponseDto.getCrDtimes() == null) {
+			abisResponseEntity.setCrDtimes(LocalDateTime.now(ZoneId.of("UTC")));
+		} else {
+			abisResponseEntity.setCrDtimes(abisResponseDto.getCrDtimes());
+		}
+		abisResponseEntity.setUpdBy(abisResponseDto.getUpdBy());
+		abisResponseEntity.setUpdDtimes(abisResponseDto.getUpdDtimes());
+		abisResponseEntity.setIsDeleted(abisResponseDto.getIsDeleted());
+		abisResponseEntity.setRespDtimes(LocalDateTime.now(ZoneId.of("UTC")));
+
+		return abisResponseEntity;
+
+	}
+
+	private AbisResponseDto updateAbisResponseEntity(AbisCommonResponseDto abisCommonResponseDto, String response) {
+		AbisResponseDto abisResponseDto = new AbisResponseDto();
+
+		abisResponseDto.setId(RegistrationUtility.generateId());
+		abisResponseDto.setRespText(response.getBytes());
+
+		abisResponseDto.setStatusCode(isInsertSuccessResponse(abisCommonResponseDto) ?
+				AbisStatusCode.SUCCESS.toString() : AbisStatusCode.FAILED.toString());
+		abisResponseDto.setStatusComment(io.mosip.registartion.processor.abis.middleware.constants.FailureReason.getValueFromKey(abisCommonResponseDto.getFailureReason()));
+		abisResponseDto.setLangCode("eng");
```

Comment:
why do we have the language hard coded?

Reply:
langcode value is not being used anywhere from abis_response table,